### PR TITLE
admin: update recommended admission controllers

### PR DESCRIPTION
**Pull request by colemickens**
Per discussion with @justinsb in Slack.

### CODE REVIEW

**Comment by colemickens** [comment]
This is what is used by the AWS bring-up: https://github.com/kubernetes/kubernetes/blob/bc382a8b0a9a7dce6f8f797f4a4499fcb7f5aaac/cluster/aws/config-default.sh#L138

File: docs/admin/admission-controllers.md [hunk 1]
```diff
@@ -139,5 +139,5 @@ Yes.
 For Kubernetes 1.0, we strongly recommend running the following set of admission control plug-ins (order matters):
 
 ```shell
---admission-control=NamespaceLifecycle,LimitRanger,SecurityContextDeny,ServiceAccount,ResourceQuota
+--admission-control=NamespaceLifecycle,LimitRanger,SecurityContextDeny,ServiceAccount,PersistentVolumeLabel,ResoureQuota
```
Comment (derekwaynecarr) [inline docs/admin/admission-controllers.md]: s/ResoureQuota/ResourceQuota

Comment (derekwaynecarr) [inline docs/admin/admission-controllers.md]: This document should have a section describing what PersistentVolumeLabel does similar to the other choices.

**Comment by kelseyhightower** [comment]
I'll fix the nit myself. LGTM

**Comment by derekwaynecarr** [comment]
Thanks Kelsey, this fell off my radar after vacation.

**Comment by colemickens** [comment]
Yes, thanks indeed, I guess I lost track of this one entirely :/.

**Pull request merged**